.github/workflows/test.yml

```diff
@@ -0,0 +1,23 @@
+name: Test Plugin
+on: [pull_request]
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Setup barebones project
+        run: |
+          mkdir test_proj && cd test_proj
+          composer init --name test/test --no-interaction --type=library --stability=dev
+          composer config repositories.wpunit-helpers '{"type": "vcs", "url": "https://github.com/pantheon-systems/wpunit-helpers.git"}'
+          composer config allow-plugins.pantheon-systems/wpunit-helpers true
+          composer require --dev pantheon-systems/wpunit-helpers:dev-${{ github.head_ref }}
+      - name: Validate that bin files were copied
+        run: |
+          set -e
+          cd ${{ github.workspace }}/test_proj
+          test -d bin || echo "❌ bin directory not found" >&2
+          test -f bin/install-wp-tests.sh || echo "❌ bin/install-wp-tests.sh not found" >&2
+          test -f bin/install-local-tests.sh || echo "❌ bin/install-local-tests.sh not found" >&2
+          test -f bin/phpunit-test.sh || echo "❌ bin/phpunit-test.sh not found" >&2
+          echo "✅ All bin files found"
```

Comment:
will these actually fail the CI if one DNE?
`||` means a non-0 exit from `test` will echo to sdterr... but will then move on without tripping `set -e`

Reply:
also it's CI, so the set -e is implicit right?

Reply:
yeah, maybe I can add exit codes there. I was sus about that when I asked chatgpt about it